libs/ledger-live-common/src/device-core/managerApi/entities/DeviceVersionEntity.ts

```diff
@@ -0,0 +1,17 @@
+import { IdEntity } from "./IdEntity";
+
+export type DeviceVersionEntity = {
+  id: IdEntity;
+  name: string;
+  display_name: string;
+  target_id: string;
```

Comment:
wouldn't be nice to set `IdEntity = string | number` and reuse it in any id field?